cypress.saucedemo.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = defineConfig({
   chromeWebSecurity: false,
   e2e: {
     baseUrl: 'https://www.saucedemo.com/',
-    excludeSpecPattern: ['cypress/e2e/homepage.cy.js', 'cypress/e2e/news.cy.js', 'cypress/e2e/spec.cy.js'],
+    excludeSpecPattern: ['cypress/e2e/homepage.cy.js', 'cypress/e2e/news.cy.js', 'cypress/e2e/spec.cy.js', 'cypress/e2e/check-news-pages.cy.js'],
```

Comment:
Just a note you can use here spec pattern for example * or ** to capture nested folders. also you could create naming schema and capture it in the pattern mobile, prod, test, env etc. Just a note no need to edit anything.